src/basic/namespace-util.c

```diff
@@ -231,3 +231,25 @@ int userns_acquire(const char *uid_map, const char *gid_map) {
         return TAKE_FD(userns_fd);
 
 }
+
+bool in_same_namespace(pid_t pid1, pid_t pid2, namespace_type ns)
+{
```

Comment:
according to our coding style, place `{` on same name as function name, i.e. remove the newline here